docs/native-modules-android.md

```diff
@@ -390,14 +577,31 @@ Another important detail to note is that a native module method can only invoke
 
 There are two approaches to error handling with callbacks. The first is to follow Node’s convention and treat the first argument passed to the callback as an error object.
 
+<Tabs groupId="android-language" defaultValue={constants.defaultAndroidLanguage} values={constants.androidLanguages}>
+<TabItem value="java">
+
 ```java
-@ReactMethod
-public void createCalendarEvent(String name, String location, Callback callBack) {
-    Integer eventId = ...
-    callBack.invoke(null, eventId);
-}
+  @ReactMethod
+   public void createCalendarEvent(String name, String location, Callback callBack) {
+       Integer eventId = ...
+       callBack.invoke(null, eventId);
+   }
 ```
 
+</TabItem>
+<TabItem value="kotlin">
+
+```kotlin
+  @ReactMethod
+  fun createCalendarEvent(name: String, location: String, callback: Callback) {
+      val eventId = ...
+      callback.invoke(null, eventId)
```

Comment:
Nothing to address here, but that's so kotlin sad and unfriendly 😅  @ShikaSD @mdvacca